### PR TITLE
update npm package versions

### PR DESCRIPTION
**Pull request by yungezz**
## Summary

update npm package versions
## Issue Type

- Bug fixing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -252,34 +252,34 @@
...
     "azure-arm-resource": "^7.3.0",
     "azure-storage": "^2.10.3",
-    "fs-extra": "^7.0.1",
+    "fs-extra": "^8.1.0",
     "fuzzaldrin-plus": "^0.6.0",
     "git-clone": "^0.1.0",
-    "ms-rest": "^2.5.0",
-    "ms-rest-azure": "^2.6.0",
+    "ms-rest": "^2.5.3",
+    "ms-rest-azure": "^2.5.5",
```
Comment (MyronFanQiu) [inline package.json]: I just want to confirm that we intend to decrease the version of ms-rest-azure. Right?

Comment (yungezz) [reply package.json]: yes, it's caused by an issue in azure-arm-resource package. azure-arm-resource 7.3.0 depends on ms-rest-azure 2.5.5, not 2.6.x.

Comment (yungezz) [reply package.json]: @MyronFanQiu fixed, could you pls review again

**Comment by MyronFanQiu** [comment]
@yungezz Should we update the version of this extention from 0.5.2 to 0.5.3 ?

**Comment by yungezz** [comment]
> @yungezz Should we update the version of this extention from 0.5.2 to 0.5.3 ?

the extension is not ready for releasing, it will bump major version if ready for release. suggest to bump version when releasing

**Pull request merged**